src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredService.test.tsx

```diff
@@ -323,7 +328,95 @@ describe('Monitored Service list', () => {
   })
 
   describe('SRM Enforcement framework tests', () => {
-    test('Should render the row switch as disabled, if the feature is disabled', async () => {
+    useFeatureFlags.mockReturnValue({ CVNG_ENABLED: true, CVNG_LICENSE_ENFORCEMENT: true })
+
+    beforeEach(() => {
+      useLicenseStoreMock.mockClear()
+    })
+
+    test('Should render the row switch as enabled, when SRM license is present and SRM_SERVICES feature is enabled, ', async () => {
+      useLicenseStoreMock.mockReturnValue(licenseWithSRMActive as unknown as useLicenseStore.LicenseStoreContextProps)
+      useFeatureFlags.mockReturnValue({ CVNG_ENABLED: true, CVNG_LICENSE_ENFORCEMENT: true })
+      const checkFeatureReturnMock2 = {
+        enabled: true
+      }
+      jest.spyOn(cvServices, 'useListMonitoredService').mockImplementation(() => ({ data: MSListData } as any))
```

Comment:
As you've used in other mocks, `mockReturnValue` is an easier way of doing this.
```suggestion
      jest.spyOn(cvServices, 'useListMonitoredService').mockReturnValue({ data: MSListData } as any)
```

Reply:
Done